### PR TITLE
Remove some no-op code in webgl.js

### LINKED ISSUE #2300: probablySupportsContext is removed from HTML and not implemented
It seems you can remove this from https://github.com/Modernizr/Modernizr/blob/master/feature-detects/webgl.js as it's not being run.

Source: https://github.com/whatwg/html/issues/3572.

### PR DESCRIPTION
**Pull request by annevk**
Fixes #2300.

### CODE REVIEW

File: feature-detects/webgl.js [hunk 1]
```diff
@@ -7,13 +7,8 @@
   "polyfills": ["jebgl", "cwebgl", "iewebgl"]
 }
 !*/
-define(['Modernizr', 'createElement'], function(Modernizr, createElement) {
+define(['Modernizr'], function(Modernizr) {
   Modernizr.addTest('webgl', function() {
-    var canvas = createElement('canvas');
-    var supports = 'probablySupportsContext' in canvas ? 'probablySupportsContext' :  'supportsContext';
```
Comment (rejas) [inline feature-detects/webgl.js]: is supportsConext also removed or why did you delete it too?

Comment (annevk) [reply feature-detects/webgl.js]: That was the original name of the feature, so yes, it's also removed: https://github.com/whatwg/html/commit/2542a12cb25ee93534cbed1f31b5e1bc05fcdd0e.
